elastic/synthetics-recorder
# [Chore] Bump deps post-v1

## Summary

Bumps dependencies for most packages. We didn't do this before v1 so as to avoid any unforeseen instability.

## Implementation details

Biggest question mark is updating `electron`, there may be some breaking change impacts as we jumped 20-24.

## How to validate this change

Make sure the recorder is working. We can do a release build to try and make sure as well.
